code/datums/emotes.dm

```diff
@@ -157,40 +157,36 @@
 		var/mob/living/L = user
 		if(HAS_TRAIT(L, TRAIT_EMOTEMUTE))
 			return FALSE
-
-/datum/emote/coders_screech //This is a shorthand to allow for custom emotes on our side while ignoring any player only checks that come with "me"
-	key = "code"
-
-/datum/emote/coders_screech/run_emote(mob/user, params, type_override, intentional = FALSE, hardcoded = FALSE) //Just override the song and dance
+/**
+* Allows the intrepid coder to send a basic emote
+* Takes text as input, sends it out to those who need to know after some light parsing
+* If you need something more complex, make it into a datum emote
+* Arguments:
+* * text - The text to send out
+*/
+/mob/proc/manuel_emote(text) //Just override the song and dance
```

Comment:
is `manuel` instead of `manual` here intended?

Reply:
Whatever do you mean?